client/authority-discovery/src/worker/schema/dht-v2.proto

```diff
@@ -18,6 +18,5 @@ message SignedAuthorityRecord {
 	bytes record = 1;
 	bytes auth_signature = 2;
 	// Even if there are multiple `record.addresses`, all of them have the same peer id.
-	// Old versions are missing this field, so `optional` will provide compatibility both ways.
-	optional PeerSignature peer_signature = 3;
+	PeerSignature peer_signature = 3;
```

Comment:
```suggestion
    // Old versions are missing this field. It is optional in order to provide compatibility both ways.
	PeerSignature peer_signature = 3;
```

Let's not remove information?

Reply:
My motivation to remove this comment was:

> Explicit 'optional' labels are disallowed in the Proto3 syntax. To define 'optional' fields in Proto3, simply remove the 'optional' label, as fields are 'optional' by default.

But sure, why not.